resources/assets/less/bem/user-home-beatmapset.less

```diff
@@ -65,8 +65,20 @@
     color: @osu-colour-f1;
   }
 
+  &__playmode-icon {
+    font-size: @font-size--normal-2;
+
+    &:not(:first-of-type) {
+      margin-left: 2px;
+    }
```

Comment:
this and the margin left on `__title` can be replaced with just margin right on `__playmode-icon`?